public/js/src/module/reasons.js

```diff
@@ -26,9 +24,10 @@ export default {
                             </div>
                         </div>
                     </header>
-                </section>` ).firstElementChild
-        );
-
+                </section>` );
+        const firstElementChild = contextual.firstElementChild || contextual.children[0];
```

Comment:
Please add comment that firstElementChild is not supported in IE11 

When ready to drop IE11, we can search the code for IE11 and clean it up.

Reply:
Got it, code updated